topic2_4/target/tomcat/conf/tomcat-users.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files